docs/user-guide/processors.md

```diff
@@ -0,0 +1,11 @@
+# Processors
+
+Processors are community packages that enable stylelint to lint the CSS within non-stylesheet files. For example, you could lint the CSS within `<style>` tags in HTML, code blocks in Markdown, or strings in JavaScript.
+
+*Processors can only be used with the CLI and the Node API, not with the PostCSS plugin.* (The PostCSS plugin will ignore them.)
+
+-   [stylelint-processor-html](https://github.com/ccbikai/stylelint-processor-html): Lint within HTML `<style>` tags.
+
```

Comment:
Can we lose the empty lines between the list item please, and order the processors alphabetically.